ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -1,7 +1,7 @@
 <svg 
   class="flight-icon flight-icon-{{@name}} {{if this.display "flight-icon-display-inline"}}"
   ...attributes
-  aria-hidden="true"
+  aria-hidden="{{if @title "false" "true"}}"
```

Comment:
I think you need to also add `aria-labelledby={{if @title this.titleId}}"` to link the title to the `svg` element, so that it is read by all screen readers. (Unless the support for the `title` element has changed since I last tested). This complexity is why I thought it would be good to include this in the component.

Reply:
nitpick: since below we have `aria-labelledby={{this.ariaLabelledby}}` would it make sense (and be more consistent) to have a similar function for this value? something like `aria-hidden={{this.ariaHidden}}`. 

(or alternatively convert line 6 to `aria-labelledby={{if @title this.titleId null}}`?)